server/historian/packages/historian-base/src/services/restGitService.ts

```diff
@@ -199,6 +199,11 @@ export class RestGitService {
         return summaryResponse;
     }
 
+    public async deleteSummary(softDelete: boolean): Promise<boolean> {
+        const headers = { "Soft-Delete": softDelete };
```

Comment:
Same comment with header. If we are always soft deleting, we don't need header. In future, if we introduce "hard-delete", we can add a new header called "Hard-Delete"